tensorflow/core/kernels/mkl/mkl_fused_ops_test.cc

```diff
@@ -39,12 +39,21 @@ namespace tensorflow {
 
 static const uint8 dummy_tensor[] = {0, 0, 0, 0, 0, 0, 0, 0};
 static const TensorShape dummy_shape({8});
+// Set the default padding value for FusedConv test.
+// Padding type will be `SAME` if padding value is INVALID_PADDING_VALUE,
+// otherwise it will be 'EXPLICIT' for Mkl ops and 'VALID' for Eigen op.
+static const int INVALID_PADDING_VALUE = -1;
```

Comment:
Per our [C++ style guide](https://google.github.io/styleguide/cppguide.html#Constant_Names), let's rename this to `kInvalidPaddingValue`.
```suggestion
static const int kInvalidPaddingValue = -1;
```